install_dependencies.sh

```diff
@@ -18,12 +18,17 @@ usage()
 }
 
 FLAVOR=`grep '^ID=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
+ID_LIKE=`grep '^ID_LIKE=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
 VERSION=`grep '^VERSION_ID=' /etc/os-release | awk -F= '{print $2}' | tr -d '"'`
 MAJOR=${VERSION%.*}
 ARCH=`uname -m`
 
-if [ $FLAVOR != "ubuntu" ]; then
-    echo "Error: Only Ubuntu is supported"
+is_ubuntu_like() {
+    [[ "$FLAVOR" == "ubuntu" || "$ID_LIKE" == *ubuntu* ]]
+}
+
+if ! is_ubuntu_like; then
+    echo "Error: Only Ubuntu and Ubuntu-based distributions are supported"
```

Comment:
```suggestion
    echo "Error: Only Ubuntu and Ubuntu-based distributions are currently supported"
```

To make it more inviting for contributions :)